app/src/androidTest/java/com/github/sdp/tarjetakuna/utils/ExportCollectionTest.kt

```diff
@@ -1,24 +1,24 @@
-package com.github.sdp.tarjetakuna.ui
+package com.github.sdp.tarjetakuna.utils

```

Comment:
not sure if it should be in utils, but maybe in a "settings" package more than utils ?

Reply:
I don't think it's related to app settings, but I'll look into changing the name later.

Reply:
yeah true, not settings, but extra features. but utils is not the right place for sure, it s supposed to be a place for code helpers